introduction.md

```diff
@@ -1 +1,4 @@
 hello, I'm already a master. Ruby is my path in codeschool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?